pkgs/development/libraries/polkit/default.nix

```diff
@@ -112,6 +117,7 @@ stdenv.mkDerivation rec {
     "--datadir=${system}/share"
     "--sysconfdir=/etc"
     "-Dsystemdsystemunitdir=${placeholder "out"}/lib/systemd/system"
+    "-Dsystemdsysusersdir=${placeholder "out"}/lib/sysusers.d"
     "-Dpolkitd_user=polkituser" #TODO? <nixos> config.ids.uids.polkituser
```

Comment:
Do we not need to drop the following now that polkit ships sysusers.d? https://github.com/NixOS/nixpkgs/blob/e16d9a1782c7e5aaba8de96cbc42d6f47ac5fd72/nixos/modules/security/polkit.nix#L110-L116

Reply:
I am not entirely sure. There is `systemd.sysusers.enable` but it's marked as experimental and is `false` by default. I don't think there is anything to change, we just ship an additional file in the package, other packages are also doing so. Unless some script scrapes packages for files in those paths? grepping for `sysusers.d` in nixpkgs doesn't yield much or much interesting stuff.

Reply:
I would expect systemd itself to use those files but maybe that is not enabled in NixOS yet. I have also seen some packages removing the file (e.g. knotdns). No idea if it is necessary.